modules/telnetd/src/main/scala/TelnetdConfiguration.scala

```diff
@@ -8,6 +8,6 @@ final case class TelnetdConfiguration(port: Int)
 object TelnetdConfiguration {
 
   val forTesting: TelnetdConfiguration =
-    apply(9091)
+    apply(9092)
```

Comment:
Why the port chang?

Reply:
For some reason on my machine 9091 didn't work and I have not yet figured out why. Any other port seems ok. I'll add a checklist above.

Reply:
So, the problem here and with empty responses on the webserver were due to another version of the app running locally in a docker container. Somehow docker snarfs traffic and routes it to containers that expose the ports, without actually binding the port locally. So you can bind the port and it won't work, but if you kill the container it will start working. Very gross. Anyway, reverting this change.

Reply:
@cquiroz sorry, I deleted the comment you responded to above. I'm not changing the ports after all. So 9090 and 9091.